packages/mobile/src/fiatExchanges/reducer.ts

```diff
@@ -5,14 +5,52 @@ import i18n from 'src/i18n'
 import { getRehydratePayload, REHYDRATE } from 'src/redux/persist-helper'
 import { RootState } from 'src/redux/reducers'
 
+export enum CiCoProvider {
+  MOONPAY = 'MOONPAY',
+  RAMP = 'RAMP',
+  SIMPLEX = 'SIMPLEX',
+  TRANSAK = 'TRANSAK',
+}
```

Comment:
nit: we have a proposed naming for enum: https://celo-org.slack.com/archives/CL6JFBALE/p1589448552015000